src/status_im/ui/screens/wallet/transactions/styles.cljs

```diff
@@ -88,3 +88,74 @@
    :height           40
    :border-radius    32
    :background-color color})
+
+;; transaction details
+
+(def transaction-details-row
+  {:flex-direction  :row
+   :margin-vertical 5})
+
+(def transaction-details-item-label
+  {:flex         1
+   :margin-right 10
+   :color        styles/color-gray4
+   :font-size    14})
+
+(def transaction-details-item-value
+  {:flex 5
+   :color     styles/color-black
+   :font-size 14})
+
+(def transaction-details-header
+  {:margin-horizontal 16
+   :margin-top        10
+   :flex-direction    :row})
+
+(def transaction-details-header-icon
+  {:margin-vertical 7})
+
+(def transaction-details-header-infos
+  {:flex            1
+   :flex-direction  :column
+   :margin-left     12
+   :margin-vertical 7})
+
+(def transaction-details-header-value
+  {:font-size   16
+   :color       styles/color-black})
+
+(def transaction-details-header-date
+  {:font-size       14
```

Comment:
Why extra spaces?

Reply:
removed longer element and forgot to clean up